modules/gitops-argo-cd-properties.adoc

```diff
@@ -40,7 +40,8 @@ a|* `<processors.operation>` - The number of operation processors.
   * `<sharding.maxShards>` - The maximum number of Argo CD Application Controller replicas.
   * `<sharding.clusterPerShard>` - The number of clusters that need to be managed by each shard. When the replica count reaches the `maxShards`, the shards manage more than one cluster.
   * `<env>` - Environment to set for the application controller workloads.
-  * `sourceNamespaces` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
+  * `<sourceNamespaces>` - The list of non-control plane namespaces for creating and managing Argo CD `Application` resources in target namespaces.
```

Comment:
@varshab1210 and @Dhruv-Soni11: FYI, L25, L26, and L43 - 1st column modified to include angular brackets. Earlier the angular brackets were missing.